tools/hardware/config.py

```diff
@@ -39,6 +47,20 @@ def get_kernel_phys_align(self) -> int:
         ''' on ARM the ELF loader expects to be able to map a supersection page to load the kernel. '''
         return self.SUPERSECTION_BITS
 
+    def align_memory(self, regions: Set[Region]) -> List[Region]:
+        ''' Arm wants physBase to be the physical load address of the kernel. '''
```

Comment:
"Arm" -> "ARM"?
```suggestion
        ''' ARM wants physBase to be the physical load address of the kernel. '''
```

Reply:
Arm is the correct spelling these days.

Reply:
It's annoying that they changed it. I'm not really keen on replacing it everywhere in a PR like this, using the old one seems wrong, and using the new one for new stuff looks inconsistent. Using the latter now and at some point doing a PR that systematically replaces ARM with Arm is probably the way to go.

Reply:
We normally choose to update it if the code is being touched.  This is the same guidelines we follow for style changes too.

Reply:
That makes sense